sdk_contrib/hapi/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "aws-xray-sdk-hapi",
+  "version": "3.1.0",
+  "description": "AWS X-Ray plugin for Hapi.JS",
+  "author": "Amazon Web Services",
+  "contributors": [
+    "Alex Coulcher<alex@polyglot.rodeo>",
+    "Eric Swann <eric.swann@q2.com>"
+  ],
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ./src",
+    "test": "mocha --recursive --exit ./test/ -R spec && tsd",
+    "test-d": "tsd"
+  },
+  "engines": {
+    "node": ">= 10.x"
+  },
+  "peerDependencies": {
+    "aws-xray-sdk-core": "^3.1.0",
+    "hapi": "*"
```

Comment:
Does this work with any version of hapi? I dunno if I've ever seen a JS library with such good backwards compatibility :D

Reply:
@anuraaga Yeah, I changed this to 17.x...I've used it with that version..don't want to go back any further than that.  Technically anything below 19 is out of maintenance at this point.